about.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Afrotech's - 2020 global Summit</title>
+    <link rel="stylesheet" href="Assets/about.css" type="text/css" />
+    <script
+      src="https://kit.fontawesome.com/02c965c448.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="mobile-menu.js" defer></script>
+  </head>
+  <body>
+    <!--Navigation-->
+    <nav>
+      <div id="top-nav">
+        <i class="fas fa-times fa-2x" id="exit-button" aria-hidden="true"></i>
+        <ul>
+          <li>
+            <a href="#"><i class="fa fa-facebook"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li><a href="#">Swahili</a></li>
+          <li><a href="#">Bemba</a></li>
+          <li><a href="#">Sign-out</a></li>
+        </ul>
+      </div>
+      <div id="bottom-nav">
+        <div id="nav-logo">
+          <i class="fa-brands fa-artstation fa-2x"></i>
+          <div>
+            <a href="./index.html">
+              <span>Afrotech-Summit</span>
+            </a>
+          </div>
+        </div>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+        </ul>
+        <div id="campaign">
+          <span>Campaign</span>
+        </div>
+      </div>
+    </nav>
+    <header class="menu">
+      <span>
+        <i class="fas fa-bars fa-2x" id="menu"></i>
+      </span>
```

Comment:
- [ ] Here on the `about.html` page, you can see your mobile menu gas issue with the `x` (close) icon. It is disappeared when the small screens size changes. I am sure you can easily fix it 👍 
<img width="623" alt="Screen Shot 2022-03-12 at 13 59 19" src="https://user-images.githubusercontent.com/49064106/158013857-8326066c-00d8-4030-bb92-42e6d0b0f6ae.png">

> The animation includes, also:

![2022-03-12 14 11 58](https://user-images.githubusercontent.com/49064106/158013933-e53655f6-223f-432f-8e80-2492603d2c9f.gif)

Reply:
Thanks working on this now